src/_pytest/nodes.py

```diff
@@ -364,6 +364,14 @@ def _repr_failure_py(
     def repr_failure(
         self, excinfo, style=None
     ) -> Union[str, ReprExceptionInfo, ExceptionChainRepr, FixtureLookupErrorRepr]:
+        """
+        Return a string representation of a collection failure.
+
+        :param excinfo: Exception information for the failure as a tuple
```

Comment:
The type of `excinfo` here is not the usual tuple, it's `ExceptionInfo[BaseException]`, which is a pytest type which wraps the tuple.

Reply:
The `_repr_failure_py()` method declares the type of `excinfo` as `ExceptionInfo[Union[Failed, FixtureLookupError]]`. Should that not be used for `repr_failure()` as well?